twilight-validate/src/request.rs

```diff
@@ -697,6 +711,11 @@ pub enum ValidationErrorType {
         /// Invalid length.
         len: usize,
     },
+    /// Provided pin limit was invalid.
+    Pin {
+        /// Invalid limit.
+        limit: i32,
```

Comment:
This should be an unsigned integer, like `u16`

Reply:
Done in https://github.com/twilight-rs/twilight/pull/2477.